.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -11,5 +11,9 @@ assignees: ''
 ----------------------------------------------------------------------------
 After deleting this template, please provide the following information:
 * Forge Version
-* *If updating was not possible*; Building Gadgets version
-* *If the issue is a crash*; An external link to the crash report
+* Building Gadgets version 
+  * Use the version Forge shows you *In-Game*
+* *If the issue is a crash*; An external link to the crash report and the **action caused crash** (e.g. update mod)
+* **How to reproduce the bug**
+* An image about the bug
```

Comment:
Could this be a little bit more specific? Reading this I'm unsure what you're referring to, which will reduce the likeliness for people to provide information.